### PR TITLE
Hide Serving CRDs in developer catalog

### PR DESCRIPTION
**Pull request by aliok**
Doing the same thing as https://github.com/openshift-knative/serverless-operator/pull/160/files, but for 1.6

### CODE REVIEW

File: olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -68,6 +68,7 @@ metadata:
       Provides a collection of API's based on Knative to support deploying and serving
       of serverless applications and functions.
     repository: https://github.com/openshift-knative/serverless-operator
+    operators.operatorframework.io/internal-objects: '["knativeservings.operator.knative.dev", "knativeservings.serving.knative.dev"]'
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml]:
```suggestion
    operators.operatorframework.io/internal-objects: '["knativeservings.operator.knative.dev"]'
```

The old group no longer exists.

**Comment by aliok** [comment]
updated and rebased

**Comment by markusthoemmes** [comment]
/test 4.4-e2e-aws-ocp-44

Setup issue. I logged the failures above as flakes in JIRA.

**Comment by markusthoemmes** [comment]
I think I'm going to just merge this. You could try rebasing again as we've put a few stabilizing patches in recently but the amount of retries you need here is ridiculous.

**Pull request merged**